src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import Display from './Display';
+import Buttonspanel from './ButtonsPanel';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
* [ ] App has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))